src/homebridge-roku.js

```diff
@@ -4,10 +4,16 @@ const DEFAULT_VOLUME_INCREMENT = 5;
 
 const { Client, keys } = require('roku-client');
 const map = require('lodash.map');
+const tlv = require('hap-nodejs/lib/util/tlv');
 
 let Service;
 let Characteristic;
 
+const DisplayOrderTypes = {
```

Comment:
Can you link me to where you found these values?

Reply:
I discovered them myself by reversing 12.2's HomeKitDaemon. I'll open an issue with my findings on KhaosT/HAP-NodeJS when I get a chance.